ext/mro/mro.xs

```diff
@@ -284,13 +289,15 @@ S_mro_get_linear_isa_c3(pTHX_ HV* stash, U32 level)
                hierarchy is not C3-incompatible */
             if(!winner) {
                 SV *errmsg;
-                Size_t i;
+                SSize_t i;
+                SSize_t count;
```

Comment:
If you're adding a new variable you can define it at first use rather than being all C89 and defining it at the top of the block.